src/server/patient/patient.config.js

```diff
@@ -1,5 +1,16 @@
 const controller = require('./controllers/patient.controller');
 
+const scopes = [
+  'user/*.*',
```

Comment:
Hmm, I feel like this would be easy to automate, with a function to which you pass `Patient` and `read` as arguments and it generates the rest of the scopes.